grakn-client/src/main/java/ai/grakn/client/Client.java

```diff
@@ -121,38 +122,58 @@ private static EngineStatus checkServerRunning() throws IOException {
      * @return true if Grakn Engine running, false otherwise
      */
     public static boolean serverIsRunning(SimpleURI uri) {
+        URL url;
         try {
-            URL url = UriBuilder.fromUri(uri.toURI()).path(REST.WebPath.KB).build().toURL();
+            url = UriBuilder.fromUri(uri.toURI()).path(REST.WebPath.KB).build().toURL();
+        } catch (MalformedURLException e) {
+            assert false : "This will never throw because we're appending a known path to a valid URI";
```

Comment:
What magic is this @aelred ?

Reply:
This is just an assertion that always fails. Basically we should never hit this code. OH I JUST REMEMBERED WE HAVE `CommonUtil.unreachableStatement()` FOR THIS